src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    public boolean isDone;
+    public String eventName;
+    public String eventType;
+
+    public Task(boolean done, String eventName) {
+        this.isDone = done;
+        this.eventName = eventName;
+        this.eventType = "T";
+    }
+
+    public Task(boolean done, String eventName, String eventType) {
+        this.isDone = done;
+        this.eventName = eventName;
+        this.eventType = eventType;
+    }
+    public void setDone() {
```

Comment:
Might be good to add an empty line between the class constructor and the method, so that there is better readability?